RobotFramework_UDS/UDSKeywords.py

```diff
@@ -213,6 +213,15 @@ def create_uds_connector(self, device_name="default", communication_name="doip",
             if isinstance(activation_type, str):
                 activation_type = int(activation_type)
 
+            if isinstance(protocol_version, str):
+                protocol_version = int(protocol_version)
```

Comment:
Hi @trimai3001 ,

It should be better if `try - except` is added to handle type conversion.

Thank you,
Ngoan